api/src/main/java/javax/enterprise/inject/spi/Annotated.java

```diff
@@ -51,18 +52,34 @@
      * 
      * @return a set of all types to which the base type should be considered assignable
      */
-    public Set<Type> getTypeClosure();
+    Set<Type> getTypeClosure();
 
     /**
      * <p>
      * Get program element annotation of a certain annotation type.
+     *
+     * Since 2.0, calling this method is considered deprecated.  To support {@see Repeatable} annotations, call {@see getAnnotation}
```

Comment:
I would use the "settled" wording (see for example `BeanManager.createInjectionTarget()`):
> This method is deprecated from CDI 2.0 and {@link #getAnnotations(Class)} should be used instead.

Also I don't think it's possible to use `{@see}` inline.